### DISCUSSION TITLE
Suddenly can't deploy anymore with "Fetching git information failed"

### DISCUSSION THREAD

**Discussion by pawlarius** [vercel/vercel#4961] [Help] [answered]
[UPDATE] Someone said use vercel-cli to deploy and it works :0

Hi all..
So previously the deployment is working fine after we pushed to master branch.
Then today I add and verified a custom domain, waits after the latest deployment is up in new domain. Checked it and it is working fine.
But when i tried to push another new commit after that, but deployment from vercel will immediately failed showing 1 single message in the log like this `14:35:38.726 | Error: Fetching git information failed`

And then i tried to push to another branch and it all failed too with the same message.
I even tried "Promote to production" from vercel dashboard with previously deployment that succeed, but never succeed, it will stuck at loading state forever :((

I checked [github status here](https://www.githubstatus.com/) and there are no issues, so i think this is probably issues from vercel.

I checked [vercel status here](https://www.vercel-status.com/), but it said All Systems Operational, meanwhile some people here also experienced the same thing out of nowhere :((
![Screen Shot 2020-08-01 at 15 24 04](https://user-images.githubusercontent.com/27823957/89097650-10cfbd00-d40b-11ea-8829-228193508b79.png)

Did anyone know what to do or have experienced this before?

**Comment by nooblyf** [comment]
I'm getting the same error
`Error: Fetching git information failed`

I did nothing but update a few dependencies.

**Reply by pawlarius** [reply]
i guess vercel is having issues then :0

**Reply by nooblyf** [reply]
yeah i think so

**Reply by pawlarius** [reply]
i checked the status of vercel here https://vercel.com/knowledge/what-is-the-vercel-platform-status, and here it stated "All Systems Operational" 😞

**Comment by lww** [comment, marked as answer]
Got the same error!
Deployment worked fine until now.

**Reply by pawlarius** [reply]
checking from their status page it said "All Systems Operational", so i tried to submit an issue with email, but using only hobby plan probably will get the replies around 2-3 days or maybe more :((

**Reply by lww** [reply]
Deployment with vercel-cli from terminal is working: `vercel --prod`

**Reply by pawlarius** [reply]
you're right, it is building now :))

**Reply by osharim** [reply]
This worked for me!

**Reply by floraCodes** [reply]
Thanksssssssss <3

**Comment by floraCodes** [comment]
Got the same issue 😫

**Comment by alanqchen** [comment]
I also have the same issue, the only thing that I changed was a GitHub workflow file.

**Reply by pawlarius** [reply]
yes i think something is wrong from vercel side, it would probably help if you have a premium plan that can directly ask their support team :"))
